pmlc/dialect/stripe/stripe_test.cc

```diff
@@ -89,13 +86,13 @@ TEST(Stripe, Transcode) {
   lbp_dev->set_name("InnerExecutor");
   lbp_dev->add_units()->set_offset(1);
   codegen::LocateBlockPass{lbp}.Apply(&cstate);
-#endif
 
   IVLOG(2, "Original version:");
   IVLOG(2, *prog->entry);
 
   IVLOG(1, "Converting to MLIR");
   auto module = IntoMLIR(&context, *prog);
+  module->dump();
```

Comment:
Please use IVLOG, there's a helper for this in `pmlc/dialect/eltwise/util.h`

Reply:
Fixed (this was just debugging code that I missed removing); thanks!